integration/grafana/README.md

```diff
@@ -23,3 +23,120 @@ is the `geopmexporter(1)`
+servers on the head node.
+(b) monitor jobs submitted to a PBS batch system.
+
+The script runs the prometheus and grafana server on the head node of a system,
+and will connect to `geopmexporter(1)` clients running on the allocated nodes 
+of a user's job.
+
+
+### Requirements
+
+A user installation of grafana and prometheus is required.  We recommend
+following the installation instructions
+[here](https://grafana.com/grafana/download?platform=linux) to `wget` the
+grafana archive for Linux.  The expanded archive is the `GRAFANA_DIR` provided
+to the `pbs_prometheus_launch.sh` script CLI.  To install prometheus download
+the archive for the latest stable Linux release from
+[here](https://prometheus.io/download/).  The expanded archive is the
+`PROMETHEUS_DIR` provided to the `pbs_prometheus_launch.sh` script CLI.
+
+Until the `geopmexporter(1)` is available in a stable release, the user must
+build and install a development snapshot of GEOPM.
+
+```bash
+    # Install latest development snapshot of GEOPM
+    GEOPM_BUILD=... # target installation directory
+    wget https://raw.githubusercontent.com/geopm/geopm/refs/heads/dev/geopmdpy/install_user.sh
+    chmod a+x install_user.sh
+    ./install_user.sh --prefix=$GEOPM_BUILD --enable-levelzero
+```
+
+
+### Configure Prometheus & Grafana
+
+Before deploying the GEOPM Prometheus exporter across the system for the
+first time, the prometheus & grafana servers need to be configured on the
+head node. The main steps to configure prometheus & grafana are:
+[A] Download and untar the prometheus & grafana builds
+[B] Configure the prometheus server with the port numbers over which
+the exported metrics are expected to stream in from the target nodes
+being monitored.
+[C] Launch the prometheus server over a desired port over which the grafana server
+is expected to query the aggregated metrics from.
+[D] Configure the port number for the grafana server over which the end user is expected
+to launch the web GUI.
+[E] Set the grafana credentials using which the end user is expected to visualize the Web UI
+[F] Download & import the grafana dashboard configuration file.
+[G] Terminate the prometheus & grafana server processes, post configuration.
+
+
+Steps-B through D can be achieved by using the `pbs_prometheus_launch.sh` script provided in
+the sane directory as this README. 
+```bash
+    # First, identify the location of prometheus & grafana builds, the
+    # preferred ports for their servers, and the grafana server admin password
+
+    PROMETHEUS_DIR=...         # Path to prometheus build on head node
+    PROMETHEUS_SERVER_PORT=... # Preferred port for the prometheus server
+    GRAFANA_DIR=...            # Path to grafana build on head node
+    GRAFANA_SERVER_PORT=...    # Preferred port for the grafana server
+
+
+    # Next, use the launch script by passing the paths & ports identified
+    # in Step-1 as parameters
```

Comment:
Where is step 1?  Was this supposed to be Step-B?

Reply:
resolved